src/components/interactive-map.js

```diff
@@ -164,6 +164,20 @@ export default class InteractiveMap extends PureComponent {
     // If props.mapControls is not provided, fallback to default MapControls instance
     // Cannot use defaultProps here because it needs to be per map instance
     this._mapControls = props.mapControls || new MapControls();
+
+    // provide an eventManager stub until real eventManager created
+    const eventManagerStub = {
```

Comment:
@Pessimistress I've been reading the code but I am unsure as to why the stub is needed? It seems like a complicated solution for the problem Shouldn't our `EventManager` class be able to handle whatever is needed here? Should we add the stub to mjolnir?

Reply:
It is because `EventManager` must be initialized with a valid DOM object. I agree that `EventManager` should be able to handle this.
Let's land this bug fix to unblock users, while I work on a more permanent solution.

Reply:
> Let's land this bug fix to unblock users, while I work on a more permanent solution.

Sounds good.